releasenotes/notes/added-sign-feature-parameterexpression-57693dd69103dc8c.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    Introduced a new feature that adds support for calculating the sign of the Parameter of the value of an expression of a Parameter class.
+    Instead of using numpy or other libraries, the user can use the instance of the ParameterExpression class to calculate the sign.
```

Comment:
```suggestion
    Instead of assigning a concrete value and using ``numpy.sign`` or other library functions, the user can use the instance of the ParameterExpression class to calculate the sign and can work with the sign before the expression is fully assigned.
```